### ISSUE TITLE
ProgressBarAndroid Component does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30941]
## Description
When using a screen reader the ProgressBarAndroid component does not announce that it is disabled.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
0.63

**Comment by Simek** [comment]
@amarlette ProgressBarAndroid component has been deprecated in the `0.63` release (https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#deprecated) and have been extracted from the core. 

Please report the issue in this repository:
* https://github.com/react-native-progress-view/progress-bar-android

**Comment by grgr-dkrk** [comment]
I attached Delegate to this component and I was able to see Talkback read aloud.
I will report some issues.

--There are no minimum or maximum props for this component. react-native-web treats `0-100` as a range.
--Android 7 and lower operating systems that do not support RangeInfo did not use TalkBack to read this percentage.
--If `indeterminate` is true and` accessibilityValue` is set, it will appear to loop through the percentages from min to max. I think It needs props to pass `accessibilityValue.text` param instead.

**Comment by mdaj06** [comment]
@amarlette can i start on this?

**Comment by cortinico** [comment]
Closing as @Simek already provided a resolution (component is deprecated and react-native-progress-view/progress-bar-android should be used instead).
